src/test/java/org/kiwiproject/dropwizard/error/ErrorContextBuilderTest.java

```diff
@@ -56,13 +65,20 @@
     private ServiceDetails serviceDetails;
     private long timeWindowAmount;
     private ChronoUnit timeWindowUnit;
+    private ScheduledExecutorService executor;
 
     @BeforeEach
     void setUp() {
         environment = DropwizardMockitoMocks.mockEnvironment();
         serviceDetails = new ServiceDetails("localhost", "127.0.0.1", 8080);
         timeWindowAmount = 20;
         timeWindowUnit = ChronoUnit.MINUTES;
+
+        executor = mock(ScheduledExecutorService.class);
+        var executorBuilder = mock(ScheduledExecutorServiceBuilder.class);
+        when(executorBuilder.build()).thenReturn(executor);
+        when(environment.lifecycle().scheduledExecutorService(any(String.class), eq(true)))
```

Comment:
Suggest changing the second matcher to `anyBoolean()` and then adding a `verify` in the appropriate tests, or maybe have a separate test that verifies we create the scheduled executor with daemon threads.